onnxruntime/python/tools/kernel_explorer/kernels/skip_layer_norm_test.py

```diff
@@ -64,7 +65,19 @@ def run_skip_layer_norm(batch_size: int, seq_len: int, hidden_size: int, dtype:
     beta_d = ke.DeviceArray(beta)
     y_d = ke.DeviceArray(output_y)
     f = getattr(ke, func)
-    my_op = f(y_d, input_d, skip_d, gamma_d, beta_d, bias_d, epsilon, hidden_size, batch_size * seq_len * hidden_size)
+
+    my_op = f(
+        y_d,
+        output_optional,
```

Comment:
Could you please add a comment here to let us know we are not test `output_optional` like `output_y `?

Reply:
will do in my next commit tomorrow, thx for the quick review

Reply:
May encounter some error if there is no `output_optional_d = ke.DeviceArray(output_optional).` and use `output_optional ` directly.

Reply:
good catch, will fix it, thx